certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
 ## 4.1 Certificate Application
 
 ### 4.1.1 Who can submit a certificate application
-In accordance with Section 5.5.2, the CA SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. The CA SHALL use this information to identify subsequent suspicious certificate requests.
+In accordance with Section 5.5.2, all CAs SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. All CA SHALL use this information to identify subsequent suspicious certificate requests.
+
+For the Root and Subordinate CAs:
+- An application for a CA certificate shall be submitted by an authorized representative of the applicant CA.
+
+For end entity certificates:
+- A certificate application shall be submitted to the CA by the Subscriber, an authorized organization representative, or an RA on behalf of the Subscriber. 
+
 
 ### 4.1.2 Enrollment process and responsibilities
-Prior to the issuance of a Certificate, the CA SHALL obtain the following documentation from the Applicant:
+For the Root and Subordinate CAs:
+- The Policy Authority is responsible for approving or denying requests for CA certificate issuances.
+
+For all CAs, prior to the issuance of any Certificate, the CA SHALL obtain the following documentation from the Applicant:
 
 1. A certificate request, which may be electronic; and
 2. An executed Subscriber Agreement or Terms of Use, which may be electronic.
 
-The CA SHOULD obtain any additional documentation the CA determines necessary to meet these Requirements.
+The certificate request SHALL contain a request from, or on behalf of, the Applicant for the issuance of a Certificate, and a certification by, or on behalf of, the Applicant that all of the information contained therein is correct.
+
+The CA SHALL be responsible for validating the information in the certificate request and the identity evidence to ensure the information is: 
 
-Prior to the issuance of a Certificate, the CA SHALL obtain from the Applicant a certificate request in a form prescribed by the CA and that complies with this CP. One certificate request MAY suffice for multiple Certificates to be issued to the same Applicant, subject to the aging and updating requirement in Section 3.3.1, provided that each Certificate is supported by a valid, current certificate request signed by the appropriate Applicant Representative on behalf of the Applicant. The certificate request MAY be made, submitted and/or signed electronically.
+- properly formed
+- accurate
+- meets the requirements for the type of certificate requested: a device Domain Validation SSL end entity certificate, a device Organizational Validation SSL end entity certificate, a CA Certificate, or a Certificate Status Server (OCSP) signing certificate
```

Comment:
- EV is already out of scope
- It's OV and DV
- Yes, we've had explicit discussions

Without debating the merits of OV versus DV as it applies to general public or commercial practices: 

- It's a very simple answer:  We have missions that want / need SUBJECT information in the certificates that explicitly assert C=US and O=U.S. Government

It all ties together with the certificate profiles and the modifications (aligned with baseline requirements) to prohibit or require subject information in the certs.  Asserting C (country) and asserting O (along with the one other required attribute for 'when O, then S/L etc') as explicit values triggers the OV policy OID  - which does not preclude automated, when properly constrained to US Government through policy **and** technical controls. 

- When you look at Section 7.1.6 for baseline requirements + subject information certificate profile requirements: it's very explicit - to assert O (organizationName), you can't assert the DV policy OID.  
- this topic has been discussed in the CA/B forum's policy working group, the March CA/B Forum F2F, and on the listserves; Taiwan brought it up in relation to the additional attributes (S/L) too specific to govt entities
- to larry's point, we dropped the OU (OU=Agency) from the profiles 

And the benefits are not on the security of the TLS transaction directly - yes, DV/OV/EV doesn't do anything different for a TLS handshake.  It's solely that for operational and informational purposes - including for mission _partners_ - we want to assert a C=US, O=U.S. Government. 

It's a government special snowflake scenario that applies to more than just US Government.  Rather than request special dispensation or exceptions to the requirements, exceptions which are hard to maintain and create too many permutations for the broader efforts in web pki, we will align.  

[The order in which we're writing _and_ identifying technical / operational design elements does have logic associated!  Section 3 has not been touched yet which will assert / document under what authority and procedures we assert C and restricted subject information aligned with the verification requirements.]